src/app/components/articles/articles.component.tsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import { useStoreon } from 'storeon/react';
-import { Article } from '../modules/articles/articles.state';
+// Modules
```

Comment:
same, comments for imports should be removed

Reply:
Comments for imports removed.